### PR TITLE
fixes issue #33 Webpage for Gaussian Data

### PR DESCRIPTION
**Pull request by mkalal6**
I've reached out to Peter, who works as Dr. McCulla's assistant, to discuss the specifics they desire for the front end and the data they need extracted from the Gaussian files. Once I receive their feedback, I'll tailor the new webpage accordingly. At present, I've set up a dedicated folder for the Gaussian webpage's dashboard and created an initial HTML file, ready for further development. Next, my plan is to design a Gaussian webpage akin to our existing ORCA page, then iterate based on the client's requirements.

### CODE REVIEW

**Comment by mkalal6** [comment]
Gaussian Webpage

The frontend now incorporates a Gaussian dashboard on the new webpage, as requested by the client to resemble the ORCA webpage. Notably, the frontend now accepts log files instead of text files. However, due to the backend's inability to process log files, this feature is currently non-functional. Nonetheless, the frontend has been adjusted to accommodate this change. To ensure smoother transitions between webpages that may be addressed in future tasks, the app.component files have reverted to their original code. This issue has been successfully resolved.

File: frontend/orca_data_converter/src/app/app-routing.module.ts
Comment (SiriChandanaGarimella) [inline frontend/orca_data_converter/src/app/app-routing.module.ts:1]: Could you please change Dashboard to Dashboard1 as it was before.

File: frontend/orca_data_converter/src/app/app.module.ts
Comment (SiriChandanaGarimella) [inline frontend/orca_data_converter/src/app/app.module.ts:1]: Rename the 'GDashboardComponent' to more meaningful variable name, something like 'GaussianDashboardComponent'. Also, 'gdashboard' to 'gaussianDashboard'

File: frontend/orca_data_converter/src/app/views/gaussianDashboard/gdashboard.component.css
Comment (SiriChandanaGarimella) [inline frontend/orca_data_converter/src/app/views/gaussianDashboard/gdashboard.component.css:1]: We have to make sure that there is code reusability. Check for the ways to make sure that ORCA css is reused here if the style is same.

**Review by SiriChandanaGarimella — COMMENTED** [review]
@mkalal6 - Please check on the review comments.

File: frontend/orca_data_converter/src/app/app.module.ts
Comment (mkalal6) [reply frontend/orca_data_converter/src/app/app.module.ts:1]: Done!

**Comment by SiriChandanaGarimella** [comment]
@mkalal6  - I committed the changes to use generic CSS and HTML templates for ORCA and Gaussian pages. Also, fixed the routing issue. Now, on hitting the URL http://localhost:4200, the ORCA dashboard (http://localhost:4200/orca) will load, and on hitting the URL http://localhost:4200/gaussian, the Gaussian dashboard will load. Could you please review and test the changes?

**Review by mkalal6 — COMMENTED** [review]
The path's work correctly when code is ran which means the reusability is successful as well as the routing. lgtm.

**Review by SiriChandanaGarimella — APPROVED** [review]
lgtm. The test case action is failing for which I will create another issue. Also, find inline comments as per client inputs.
Client's Request for Gaussian Webpage
"The ORCA and Gaussian websites should be very similar or perhaps even merged if possible. - **Will create another issue for this point**
The types of data needed are going to be very similar, the only difference will be the formatting of the text in the output files. - **Need to get more details on this point**
The suggestion would be to try and duplicate the orca website with the gaussian website. - **Done**
It might be easier to select the output file type (ORCA or Gaussian) on the same page, and then use the same code to do the text extraction. - **will create another issue for this point**"

**Pull request merged**